### PR TITLE
Added MySql driver

### PR DESCRIPTION
**Pull request by roeepolegfiverr**

### CODE REVIEW

File: shared/connectors.go [hunk 1]
```diff
@@ -108,3 +129,28 @@ func createMongoClient(clientName string) (*mgo.Session, error) {
...
+
+func createMySqlClient(clientName string) (client *sql.DB, err error) {
+	user := Config.Get(fmt.Sprintf("mysql.%s.host", clientName), "root")
+	password := Config.Get(fmt.Sprintf("mysql.%s.host", clientName), "")
+	host := Config.Get(fmt.Sprintf("mysql.%s.host", clientName), "localhost")
+	port := Config.GetInt(fmt.Sprintf("mysql.%s.port", clientName), 3306)
+	dbName := Config.Get(fmt.Sprintf("mysql.%s.port", clientName), "fiverr_dev")
+
+	fmt.Println("connecting:" + fmt.Sprintf("%s:%s@tcp(%s:%d)/%s", user, password, host, port, dbName))
+	client, err = sql.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s:%d)/%s", user, password, host, port, dbName))
```
Comment (nitsanfiverr) [inline shared/connectors.go]: don't you want to save the sprintf to a var ? why call it twice ?

File: shared/connectors.go [hunk 2]
```diff
@@ -108,3 +129,28 @@ func createMongoClient(clientName string) (*mgo.Session, error) {
...
+		log.Print(err)
+		return nil, err
+	}
+	// TODO: add close to
+	// defer client.Close()
+
+	// Open doesn't open a connection. Validate DSN data:
+	err = client.Ping()
+	if err != nil {
+		panic(err.Error()) // proper error handling instead of panic in your app
```
Comment (nitsanfiverr) [inline shared/connectors.go]: comment you copied ?

File: shared/connectors.go [hunk 3]
```diff
@@ -108,3 +129,28 @@ func createMongoClient(clientName string) (*mgo.Session, error) {
...
+	if err != nil {
+		fmt.Println(err)
+		log.Print(err)
+		return nil, err
+	}
+	// TODO: add close to
+	// defer client.Close()
+
+	// Open doesn't open a connection. Validate DSN data:
+	err = client.Ping()
```
Comment (nitsanfiverr) [inline shared/connectors.go]: why the ping ? is is necessary ?

File: shared/connectors.go (hunk 1 above)
Comment (roeepolegfiverr) [inline shared/connectors.go]: just for debug, i'll remove it later when we're stable.

File: shared/connectors.go (hunk 2 above)
Comment (roeepolegfiverr) [inline shared/connectors.go]: no

File: shared/connectors.go (hunk 3 above)
Comment (roeepolegfiverr) [inline shared/connectors.go]: it's the only way to test it's working before any real code starts

**Pull request closed without merging**